### PR TITLE
Add PyInventory into apps.json

### PR DESCRIPTION
**Pull request by jedie**
https://github.com/jedie/pyinventory_ynh
https://github.com/jedie/PyInventory

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -2522,6 +2522,16 @@
...
+    "pyinventory": {
+        "branch": "main",
+        "category": "productivity_and_management",
+        "revision": "HEAD",
+        "state": "inprogress",
+        "subtags": [
+            "business_and_ngos"
+        ],
+        "url": "https://github.com/jedie/pyinventory_ynh"
+    }
```
Comment (kay0u) [inline apps.json:2534]:
```suggestion
    },
```

**Review by kay0u — APPROVED** [review]
:+1:

**Review by ericgaspar — APPROVED** [review]

**Comment by jedie** [comment]
should i rebase/squash or are two commits okay?

**Comment by kay0u** [comment]
No it's ok, thank you for this app! :100:

**Pull request merged**